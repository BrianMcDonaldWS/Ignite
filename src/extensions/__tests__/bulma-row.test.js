import MarkdownIt from 'markdown-it';
import row from '../bulma-row';

const md = new MarkdownIt().use(row);

test('bulma - row', () => {
  expect(
    md.render(`
||| row
# Title

## I can contain various combinations of elements.
|||
  `)
  ).toMatchSnapshot();
});
