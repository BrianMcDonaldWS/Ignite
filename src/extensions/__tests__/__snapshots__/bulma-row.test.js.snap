// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - row 1`] = `
"
        <div class=\\"columns \\">
      <h1>Title</h1>
<h2>I can contain various combinations of elements.</h2>

      </div>
    "
`;
